features/docs/orca_automation_sources.feature.md

```
@@ -0,0 +1,53 @@
+
+## Scenario: README は issue coordinator の automation id を示す
+
+- Then リポジトリファイル "docs/agents/automations/README.md" は "1c20e813-8150-4f46-a44a-46cea275af4c" を含む
+
+## Scenario: README は PR reviewer の automation id を示す
+
+- Then リポジトリファイル "docs/agents/automations/README.md" は "b467ad1d-295a-4146-ab47-b1fe2636ddc3" を含む
+
+## Scenario: README は実行スケジュールを示す
+
+- Then リポジトリファイル "docs/agents/automations/README.md" は "*/10 * * * *" を含む
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)